### PR TITLE
Make README less cluttered

### PR DESCRIPTION
**Pull request by mefinity**
Changes:
Redo the look for Badge
Stop using large social media banners
Put DPI notice in a dropdown

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,254 +1,149 @@
...
+<a href="https://trakteer.id/bebasidbykini">
+    <img alt="Static Badge" src="https://img.shields.io/badge/donasi-white?style=for-the-badge&logo=buymeacoffee&logoColor=white&label=Trakteer&labelColor=222">
+</a>
+
+<a href="https://saweria.co/bebasid">
+    <img alt="Static Badge" src="https://img.shields.io/badge/donasi-white?style=for-the-badge&logo=iconjar&logoColor=white&label=Saweria&labelColor=222">
+</a>
 </p>
 
+## Sponsor
```
Comment (ryukora) [inline README.md:44]: It should be ``Sponsors``

File: README.md [hunk 2]
```diff
@@ -1,254 +1,149 @@
...
+<a href="LICENSE">
+    <img alt="Static Badge" src="https://img.shields.io/badge/MIT-white?style=for-the-badge&logo=andela&logoColor=white&label=Lisensi&labelColor=222">
+</a>
+
+<img alt="Static Badge" src="https://img.shields.io/badge/2019-white?style=for-the-badge&logo=fireship&logoColor=white&label=Diluncurkan&labelColor=222">
+
 <br>
 
-<p align="center">
-<code>Server Komunitas:</code></br>
```
Comment (ryukora) [inline README.md:92]: Maybe try adding it back, but putting back on the hidden. on contact. Like click this button to view for community server

**Review by ryukora — COMMENTED** [review]
Try improve something, but putting back on behind.

File: README.md [hunk 3]
```diff
@@ -1,254 +1,149 @@
...
-    <m>Dimulai dari <b>Rp. 50.000,00</b></m>.</br>
-    <a href="https://www.atharva.cloud/index.php?rp=/store/cloud-vm-lite">Lihat paket langganan VPS selengkapnya</a>.
-</div>
-<hr>
-<p align="center"><code>KINI Memperkenalkan:</code></br></br>
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-black.svg#gh-light-mode-only" alt="bebasID White Logo" width="445">
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo.svg#gh-dark-mode-only" alt="bebasID Black Logo" width="445"></br>
-    <i>Membantumu berselancar dengan bebas dan tak ada habisnya!</i>
-</p>
+#### Bahasa / Language
```
Comment (ryukora) [inline README.md:3]: Add the tables for this title, make it to group

**Comment by ryukora** [comment]
It look better, but you have choice to do before doing it again

File: README.md (hunk 1 above)
Comment (mefinity) [reply README.md:44]: Atharva is currently the only sponsor, which is singular

File: README.md (hunk 2 above)
Comment (mefinity) [reply README.md:92]: I dont think a dropdown would look good squished between repo info and donation badges

File: README.md (hunk 3 above)
Comment (mefinity) [reply README.md:3]: I dont think it would add anything to the look, and it would make the raw file look more cluttered than it already is

**Review by ryukora — APPROVED** [review]
Ok, i'll try commit it right now.

**Pull request merged**